.gitignore

```diff
@@ -27,3 +27,6 @@
 
 bin*
 dist/
+
+sample.yaml
```

Comment:
I didn't find a sample.yaml file in the commits.

Reply:
Because of the ignore.